src/pages/en/guides/markdown-content.md

```diff
@@ -395,9 +424,14 @@ We also suggest [diving into their theme documentation](https://github.com/shiki
 #### Prism configuration
 
 When using Prism, you'll need to add a stylesheet to your project for syntax highlighting. If you're just getting started and prefer to use Prism over Shiki, we suggest:
+
 1. [Setting `syntaxHighlight: 'prism'`](#choose-a-syntax-highlighter) from your `@astrojs/markdown-remark` config.
 1. Choosing a premade stylesheet from the available [Prism Themes](https://github.com/PrismJS/prism-themes).
-2. Adding this stylesheet to [your project's `public/` directory](https://docs.astro.build/en/core-concepts/project-structure/#public).
-3. Loading this [into your page's `<head>`](https://docs.astro.build/en/core-concepts/astro-pages/#page-html) via a `<link>` tag.
+1. Adding this stylesheet to [your project's `public/` directory](https://docs.astro.build/en/core-concepts/project-structure/#public).
+1. Loading this [into your page's `<head>`](https://docs.astro.build/en/core-concepts/astro-pages/#page-html) via a `<link>` tag.
 
 You can also visit the [list of languages supported by Prism](https://prismjs.com/#supported-languages) for options and usage.
+
+## Notes
+
+- Astro will add autogenerated ids to all the headings automatically using [github-slugger](https://github.com/Flet/github-slugger). If a custom id is specified, it won't be overriden.
```

Comment:
Not sure if this clarification is necessary given this is the Markdown page, but probably wouldn’t hurt.

```suggestion
- Astro will add autogenerated ids to all headings in Markdown files automatically using [github-slugger](https://github.com/Flet/github-slugger). If a custom id is specified, it won't be overriden.
```